content/news/2018/05/brand-bible-new-merchants/index.md

```diff
@@ -0,0 +1,84 @@
+hash512 (on Discord) reports that he’s building a NavTV box as a more powerful alternative to the NavPi. Get hold of him on Discord if you’d like to help out with this project by collaborating on communications, website development or wallet integration.
+
+Prodpeak (on Discord) is also working on a block explorer. If you’d like to give him a hand, reach out on Discord. There are plenty of technical challenges for smart devs to overcome, and as they say ‘many hands make light&nbsp;work.’
+
+### Content and Media
+One of the key focuses for the NavCoin Core team over next few months is in creating a framework that makes it easy for the wider NavCoin community to contribute. Whether that’s creating content, organising meetups, or developing tools - it should be clear how anyone can get involved and the steps involved. It won’t be the NavCoin core team telling everyone what they should be working on, but rather a much more organic approach where individuals decide what they want to work on. It’s less about having a large single team that everyone joins, and more about having lots of smaller independent teams that people can easily move between.
+
+The first part of this framework is the NavCoin brand guidelines (known as the bible) and the Developer and Content Creator manifesto’s.
+
+<br />
+<section style="text-align: center">
+![](img5.png)
+<br /><br />
+</section>
+
+The brand bible sets out the Core Media and Content Creators suggestions on typefaces and how to use them, it displays the brand’s colour palate, and gives useful tips on how to create visuals and write with the NavCoin tone of voice. It’s hoped that having a brand bible will make it easier for you to get started on creating your own content for NavCoin. It’s almost finished, and will be available to download soon from the NavCoin Community&nbsp;Hub.
+
+The content collective have also been refining the ‘Core Developer’, and ‘Core Media and Content Creator’ manifestos. These documents detail the beliefs NavCoin core agree to uphold as members of the NavCoin community. Once they’re completed they will also be available on the community site. Look out for them soon.
+
+The NavCoin Core team are also looking to rename the NavCore community hub website to make it more clear what the purpose of the website is. NavCoin.org is aiming to be information related to the protocol - how it works, the technical information, and documentation. The community hub (NavCore.org) on the other hand is more focused on community efforts - such as which projects are being worked on, and how to get involved. So please submit your ideas for domain names to our Discord or Reddit channels - the only requirement is that the domains are available for use.
```

Comment:
`So please submit your ideas for domain names to our Discord or Reddit channels`

We should probably avoid the use of the word `so` where we can.